metafix/src/main/java/org/metafacture/metafix/Value.java

```diff
@@ -705,45 +533,14 @@ public Value append(final String fieldPath, final String newValue) {
          */
         public Value get(final String field) {
             // TODO: special treatment (only) for exact matches?
-            final List<Value> list = findFields(field).map(map::get).collect(Collectors.toList());
+            final List<Value> list = matchFields(field, Stream::filter).map(map::get).collect(Collectors.toList());
```

Comment:
Why did you drop the `findFields()` helper?

Reply:
I mostly tried to trim down the similar named methods, so I dropped one-liners. Feel free to add anything back that you feel is an improvement.